src/Command/PopulateCommand.php

```diff
@@ -139,7 +139,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $options['last_page'] = $input->getOption('last-page');
         }
 
-        if ($input->isInteractive() && $reset && 1 < $input->getOption('first_page')) {
+        if ($input->isInteractive() && $reset && 1 < $input->getOption('first-page')) {
```

Comment:
You stored the value of `$input->getOption('first-page')` in `$options['first_page']`. 
Why not re-use that value ?

Reply:
A good point, I changed it.